binnary.html

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <link rel="stylesheet" href="assets/styles/binnary.css"/>
+    <title>Title</title>
+</head>
+<body>
+<input placeholder="input value" id="value"/>
```

Comment:
_UI_: add tips/placeholder so the user can understand how the input data should look like. I can't understand at the first time how my input string should look like

Reply:
fixed